### PR TITLE
feature/CVSB-489

### PR DESCRIPTION
**Pull request by mdclarke21**
UI for 'Additional details' page (https://marvelapp.com/f9g6534/screen/50454285)

New pages added to accommodate the new design since alpha:
- Brakes
- Weights
- Tyres
- Additional details

### CODE REVIEW

File: src/pages/testing/test-creation/vehicle/vehicle-tyres/vehicle-tyres.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  constructor(public navCtrl: NavController, public navParams: NavParams, public viewCtrl: ViewController) {
+    this.viewCtrl = viewCtrl;
+  }
+
+  ionViewWillEnter() {
+    this.viewCtrl.setBackButtonText('Vehicle details');
+  }
+
+  ionViewDidLoad() {
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-tyres/vehicle-tyres.ts]: remove lifecycle hook and console.log

File: src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                <h3>Kerb</h3>
+            </ion-label>
+            <div item-content>
+                <p>0</p>
+            </div>
+        </ion-item>
+    </ion-list>
+</ion-content>
+  
+<ion-footer>
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.html]: remove element if not used

File: src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams, ViewController } from 'ionic-angular';
+
+/**
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts]: remove comments

File: src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+  constructor(public navCtrl: NavController, public navParams: NavParams, public viewCtrl: ViewController) {
+    this.viewCtrl = viewCtrl;
+  }
+
+  ionViewWillEnter() {
+    this.viewCtrl.setBackButtonText('Vehicle details');
+  }
+
+  ionViewDidLoad() {
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts]: remove hook and console.log

File: src/theme/variables.scss [hunk 5]
```diff
@@ -43,18 +52,29 @@ $colors: (
...
+$content-background-color: #EFEFF4; 
+$reg-plate-color: #FFD84C;
 
 // App iOS Variables
 // --------------------------------------------------
 // iOS only Sass variables can go here
 
-$list-ios-header-background-color: #F5F5F5;
-$list-ios-border-color: #c8c7cc;
+$list-ios-header-background-color: #EFEFF4;
```
Comment (danieliordache) [inline src/theme/variables.scss]: Those variables are used by ionic for every list present into the app. Have you checked if other pages containing lists are not affected?

Comment (mdclarke21) [reply src/theme/variables.scss]: I think this is ok. I updated the colours based on the marvel app prototype.

The 'Search for an ATF' list uses the `ion-item-divider` element for the letters A B, C etc. The lists I am working on use the `ion-list-header` element.

File: src/app/app.scss [hunk 6]
```diff
@@ -15,39 +15,50 @@
 // automatically applied to the <body> element in the app.
 
 .header-ios {
+  // height: $header-height-default;
```
[the remark this answers is not in the document]
Comment (mdclarke21) [reply src/app/app.scss]: Done

File: src/app/app.scss [hunk 7]
```diff
@@ -15,39 +15,50 @@
...
     min-height: 80px;
-
+    height: inherit;
+    background: $black;
+    
     ion-title {
       padding: 0;
       height: 44px;
-      margin-left: 10px;
+      margin-left: 14px;
```
[the remark this answers is not in the document]
Comment (mdclarke21) [reply src/app/app.scss]: Done

File: src/app/app.scss [hunk 8]
```diff
@@ -15,39 +15,50 @@
...
       margin-top: 55px;
 
       .toolbar-title-ios {
         text-align: left;
-        font-size: 28px;
-        color: #fff;
+        font-size: $font-size-large;
+        font-weight: $font-weight-bold;
+        line-height: 41px;
+        color: $white;
```
[the remark this answers is not in the document]
Comment (mdclarke21) [reply src/app/app.scss]: Done

File: src/pages/testing/test-creation/vehicle/vehicle-additional/vehicle-additional.ts [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams, ViewController } from 'ionic-angular';
+
+/**
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-additional/vehicle-additional.ts]: still present

File: src/pages/testing/test-creation/vehicle/vehicle-brakes/vehicle-brakes.ts [hunk 10]
```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams, ViewController } from 'ionic-angular';
+
+/**
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-brakes/vehicle-brakes.ts]: comment still present

File: src/pages/testing/test-creation/vehicle/vehicle-tyres/vehicle-tyres.ts [hunk 11]
```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams, ViewController } from 'ionic-angular';
+
+/**
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-tyres/vehicle-tyres.ts]: comments still present

File: src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts [hunk 12]
```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams, ViewController } from 'ionic-angular';
+
+/**
```
Comment (danieliordache) [inline src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts]: comments still present

File: src/pages/testing/test-creation/vehicle/vehicle-additional/vehicle-additional.ts (hunk 9 above)
Comment (mdclarke21) [reply src/pages/testing/test-creation/vehicle/vehicle-additional/vehicle-additional.ts]: Done

File: src/pages/testing/test-creation/vehicle/vehicle-tyres/vehicle-tyres.ts (hunk 11 above)
Comment (mdclarke21) [reply src/pages/testing/test-creation/vehicle/vehicle-tyres/vehicle-tyres.ts]: Done

File: src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts (hunk 12 above)
Comment (mdclarke21) [reply src/pages/testing/test-creation/vehicle/vehicle-weights/vehicle-weights.ts]: Done

File: src/pages/testing/test-creation/vehicle/vehicle-brakes/vehicle-brakes.ts (hunk 10 above)
Comment (mdclarke21) [reply src/pages/testing/test-creation/vehicle/vehicle-brakes/vehicle-brakes.ts]: Done

**Comment by tealorg** [comment]
Done

**Comment by tealorg** [comment]
Please resolve conflicts, it won't merge, thanks

**Pull request merged**